src/couch/src/couch_doc.erl

```diff
@@ -270,6 +270,9 @@ transfer_fields([{<<"_replication_state">>, _} = Field | Rest],
 transfer_fields([{<<"_replication_state_time">>, _} = Field | Rest],
     #doc{body=Fields} = Doc) ->
     transfer_fields(Rest, Doc#doc{body=[Field|Fields]});
+transfer_fields([{<<"_replication_start_time">>, _} = Field | Rest],
```

Comment:
Oh interesting. It's the case of replicator the _replicator db itself right? 

Anyone have any thoughts on this one  @sagelywizard @rnewson ?

Add a note that replicating the _replicator from CouchDB 3.0 to <3.0 is not supported if it has completed replications in the source db?

Use a non-underscored version? `replication_start_time`. But that's inconsistent a bit with other ones.

Remove this field? But then _scheduler/docs docs output will be inconsistent since for completed replications we read that from the view on disk. Dashboard code will have to be modified as well perhaps.